config/locales/server.en.yml

```diff
@@ -868,6 +868,9 @@ en:
     you_must_edit: '<p>Your post was flagged by the community. Please <a href="%{path}">see your messages</a>.</p>'
     user_must_edit: "<p>This post was flagged by the community and is temporarily hidden.</p>"
 
+  ignored:
+    hidden_content: '<i>Hidden content</i>'
```

Comment:
I think it would be better to remove the formatting from the string and instead add a class on the post saying it has hidden content so that we can style it properly using CSS

Reply:
> add a class on the post saying it has hidden content

How do you do that? Does adding a class occur on the backend?

Reply:
Maybe you meant:

`<p class="hidden-content">.....</p>`

Reply:
Just wrapping up our offline discussion, we thought it would be better to:

1. Expose `ignored` attribute on the PostSerializer
2. Let the front-end handle the state of an ignored post